lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -150,18 +159,7 @@ def connection
       end
 
       def auth_token
-        @auth_token ||= begin
-          client = Signet::OAuth2::Client.new(
-            token_credential_uri: TOKEN_CREDENTIAL_URI,
-            client_id: Fastlane::Actions::FirebaseAppDistributionLoginAction::CLIENT_ID,
-            client_secret: Fastlane::Actions::FirebaseAppDistributionLoginAction::CLIENT_SECRET,
-            refresh_token: ENV["FIREBASE_TOKEN"]
-          )
-          client.fetch_access_token!
-          return client.access_token
-        rescue Signet::AuthorizationError
-          UI.crash!(ErrorMessage::REFRESH_TOKEN_ERROR)
-        end
+        @auth_token ||= fetch_auth_token("")
```

Comment:
Instead of passing an empty string here consider adding a default parameter to the `fetch_auth_token` method e.g.

```
def fetch_auth_token(service_credentials_file_path=nil)
...
end
```